package-lock.json

```diff
@@ -5,12 +5,12 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.12.13",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.13.tgz",
-      "integrity": "sha512-HV1Cm0Q3ZrpCR93tkWOYiuYIgLxZXZFVG2VgK+MBWjUqZTundupbfx2aXarXuw5Ko5aMcjtJgbSs4vUGBS5v6g==",
+      "version": "7.12.11",
```

Comment:
Why I still see package-lock changes here?

Reply:
🤔 From where should I revert the package-lock? Should I use the current one from svgo or from a previous commit of this branch? `npm install` seems to update the package lock on its own.

Reply:
git checkout HEAD~48 -- package-lock.json